### PR TITLE
Use vercel/pkg to generate single binaries

### LINKED ISSUE #332: Use vercel/pkg to create Monika binary

### PR DESCRIPTION
**Pull request by dennypradipta**
# Monika Pull Request (PR)  

## What feature/issue does this PR add  
This PR fixes #332 

## How did you implement / how did you fix it  
I removed the libraries that uses external binaries such as `open`, and created a script to package the app as binary using `npm run pkg`

## How to test  
1. Run `npm run pkg`
2. Go to the dist folder
3. Run the binary according to your operating system without any config
4. Run the binary according to your operating system with a config copied to the dist folder

### CODE REVIEW

**Review by gsukanto — APPROVED** [review]
lgtm

File: package.json [hunk 1]
```diff
@@ -21,13 +21,14 @@
...
-    "start": "./bin/run",
+    "prepack": "rm -rf lib tsconfig.tsbuildinfo && tsc -b && oclif-dev readme",
+    "start": "npm run prepack && ./bin/run",
     "test": "cross-env NODE_ENV=test nyc --extension .ts mocha --forbid-only \"{src,test}/**/*.test.ts\"",
     "prod_test": "cross-env NODE_ENV=test nyc --extension .ts mocha --forbid-only \"prod_test/**/*.test.ts\"",
     "version": "oclif-dev readme && git add README.md",
     "format": "prettier --write .",
-    "check-format": "prettier --check ."
+    "check-format": "prettier --check .",
+    "pkg": "npm run prepack && pkg --out-path dist/ . --compress brotli"
```
Comment (ariya) [inline package.json:31]: `out-path` is not necessary since it's already specified in `pkg` field in this JSON file.

Comment (dennypradipta) [reply package.json]: Whoopsie, should be fixed in the latest commit!

**Review by ariya — APPROVED** [review]
:100:

**Pull request merged**